core/domain/html_cleaner.py

```diff
@@ -458,13 +460,127 @@ def convert_to_text_angular(html_data):
     return unicode(soup).replace('<br/>', '<br>')
 
 
-def convert_tag_contents_to_text_angular(html_data):
+def convert_to_ck_editor(html_data):
+    """This function converts the html to CKEditor supported format.
```

Comment:
Minor: `...converts html strings to CKEditor supported format`

Reply:
Done!